packages/graphql/tests/tck/union.test.ts

```diff
@@ -28,7 +28,7 @@ describe("Cypher Union", () => {
     let typeDefs: DocumentNode;
     let neoSchema: Neo4jGraphQL;
 
-    beforeAll(() => {
+    beforeEach(() => {
```

Comment:
Any particular reason? 👀 
```suggestion
    beforeAll(() => {
```

Reply:
because some tests use the experimental flag set to true while others don't

Reply:
Ah I see, I didn't catch it!